pkg/image-manager/handlers.go

```diff
@@ -65,7 +65,6 @@ func baseImageEntityToModel(e *BaseImage) *models.BaseImage {
 	m := models.BaseImage{
 		CreatedTime: e.CreatedTime.Unix(),
 		DockerURL:   swag.String(e.DockerURL),
-		Language:    models.Language(e.Language),
```

Comment:
I'm confused.  How are we conveying the language of the [base]image back to the user?